### ISSUE TITLE
Support dynamically created horizontal ListView

### ISSUE THREAD

**Issue by YohanWadia** [flutter/flutter#26803]
Ive noticed a lot of people struggle with a horizontal list view that is made dynamically, it would be really nice to see how that is done.
Also a rating widget where you can rate how many stars you want to give something

**Comment by Kartik1607** [comment]
If I remember correctly a listview contains scroll direction property and you can change that to get horizontal list. 

For rating widget you can simply use a row widget and pass rating icons.

**Comment by YohanWadia** [comment]
@Kartik1607 , Yes ListView does contain Horizontal scrolling, but for some reason something goes wrong with the height. Well I've seen some tutorials too, but some developers too, like me arent very convinced. SO I wanted to request one in the Flutter App, **cause the example they have in the App, at Present, only shows the circular leading letters as a large Icon!**

And for the the rating app, I finally did do what you said! ;-)  
It worked fine.
Thanks

**Comment by Kartik1607** [comment]
Hi, it would be nice to know what kind of examine app you want. What is the desired end result. I think it would be nice to provide advanced sample as well.

**Comment by zoechi** [comment]
As @Kartik1607 pointed out this is not enough information.

Please consider asking support questions in one of the other channels listed at http://flutter.io/support .

**Comment by YohanWadia** [comment]
@zoechi ... on the App it self I tapped on FeedBack .. but it brought me here!
I did look for other contacts but found none.

**Comment by YohanWadia** [comment]
Yeah...
@Kartik1607  .. I will post it there.
but here is what I meant.
I tried to push the planets into a horizontal scrolling listView. 
https://sergiandreplace.com/planets-flutter-creating-a-list-of-planets/

SO, I checked the app, but it doesnt show much either, so thought we should have something like this.. thats why I requested it.

I've asked other developers too, and most of them couldnt solve the Horizontal Scrolling ListView holding children of something like a ListTile/Card.

In android I could have pulled it across  with Fragments & PageView, but in Flutter its kinda not happening.
